visualdl/logic/sdk.cc

```diff
@@ -128,7 +146,16 @@ std::vector<int> ScalarReader<T>::ids() const {
 template <typename T>
 std::vector<time_t> ScalarReader<T>::timestamps() const {
   std::vector<time_t> res;
-  for (int i = 0; i < reader_.total_records(); i++) {
+  for (size_t i = 0; i < reader_.total_records(); ++i) {
```

Comment:
I think we can unify this two interface, use default parameter zero.
`timestampes(size_t start_index = 0) const `